antarest/study/storage/rawstudy/watcher.py

```diff
@@ -129,7 +130,17 @@ def _rec_scan(
                     f"No scan directive file found. Will skip further scan of folder {path}"
                 )
                 return []
+
+            study_antares_exist: bool = False
+
             if (path / "study.antares").exists():
+                study_antares_exist = True
+            elif path.suffix == ".zip":
+                zf = ZipFile(path, "r")
+                if str(Path(path.stem) / "study.antares") in zf.namelist():
+                    study_antares_exist = True
```

Comment:
don't try to sync archived studies (only managed archived studies will be handled)